Ajoute 9 nouveaux articles à la documentation

bboulesteix opened the pull request:
Comme évoqué dans #36, cette _pull-request_ ajoute de nouveaux articles à la documentation.

Articles ajoutés :

1. [a-propos/signaler-un-contenu.md](https://github.com/etalab/doc.data.gouv.fr/pull/52/files#diff-9bea2554a0def95606b76a2e0ee9a7a1)
2. [jeux-de-donnees/demander-a-datagouvfr-de-moisonner-votre-site.md](https://github.com/etalab/doc.data.gouv.fr/pull/52/files#diff-bb07e1082b07bad6ab116d9fdff312d9)
3. [jeux-de-donnees/faire-une-recherche.md](https://github.com/etalab/doc.data.gouv.fr/pull/52/files#diff-708b525a435336fb50560f3c3fb82989)
4. [organisations/ajouter-un-utilisateur-a-une-organisation.md](https://github.com/etalab/doc.data.gouv.fr/pull/52/files#diff-dad53194444d7a5cbb12c19e9fb6571e)
5. [organisations/demander-a-rejoindre-une-organisation.md](https://github.com/etalab/doc.data.gouv.fr/pull/52/files#diff-aa4cd4eb5763797db7cedbd0c67577b1)
6. [organisations/differences-administrateur-editeur-organisation.md](
https://github.com/etalab/doc.data.gouv.fr/pull/52/files#diff-f0ccf2441ef829270ebc752b0d90e033)
7. [organisations/retirer-un-utilisateur-d-une-organisation.md](https://github.com/etalab/doc.data.gouv.fr/pull/52/files#diff-dad53194444d7a5cbb12c19e9fb6571e)
8. [reutilisations-et-discussions/moderer-une-discussion.md](
https://github.com/etalab/doc.data.gouv.fr/pull/52/files#diff-dc837e2c6d7d4f8dfc4b0dac8ba3c9c8)
9. [reutilisations-et-discussions/modifier-ou-supprimer-une-reutilisation.md](
https://github.com/etalab/doc.data.gouv.fr/pull/52/files#diff-94cfebf448d3661edbeca12c6c04d3b5)

bboulesteix committed:
Ajoute un article sur le signalement de contenus

Ce commit ajoute un article dans la section À propos.

bboulesteix committed:
Ajoute deux articles consacrés aux jeux de données

Ce commit ajoute les articles :
1. `jeux-de-donnees/faire-une-recherche.md`
2. `jeux-de-donnees/demander-a-datagouvfr-de-moisonner-votre-site.md`

bboulesteix committed:
Ajoute quatre articles consacrés aux organisations

Ce commit ajoute 4 articles à la section /organisations de la documentation.

bboulesteix committed:
Ajoute deux articles consacrés aux réutilisations

abulte commented on articles/_jeux-de-donnees/demander-a-datagouvfr-de-moisonner-votre-site.md:
```diff
@@ -0,0 +1,100 @@
+---
+# Demander à data.gouv.fr de moissonner votre site
+
+Si vous mettez en ligne des données publiques sur une plateforme ouverte, dans un format dont les métadonnées correspondent à la syntaxe [ODS](https://www.opendatasoft.com), [CKAN](https://ckan.org), [DCAT](https://www.w3.org/TR/vocab-dcat/), ou MAAF (agriculture), vous pouvez les référencer _automatiquement_ sur data.gouv.fr en utilisant notre service de moissonnage.
```
s/agriculture/Ministère de l'Agriculture et de l'Alimentation/ peut-être.

💡préciser que le moissonneur `maaf` est spécifique, voire ne pas en parler tellement il est spécifique 🤔

bboulesteix replied:
Noté. Dans ce cas, précisions qu’il s’agit d’un moissonneur spécifique. Ne pas en parler pourrait être interprété comme un oubli, ce qui me dérangerait.

abulte commented on articles/_jeux-de-donnees/demander-a-datagouvfr-de-moisonner-votre-site.md:
```diff
@@ -0,0 +1,100 @@
+
+Le service de moissonnage permet de référencer sur data.gouv.fr des jeux de données publiés ailleurs sur le web. De cette manière, vous n’avez pas besoin d’importer à la main sur data.gouv.fr les jeux de données que vous avez déjà importés sur votre propre plateforme.
+
+Un seul moissonneur permet d’importer toutes les données d’un portail d’open data. Il n’est pas nécessaire de créer un moissonneur par jeu de données à importer sur data.gouv.fr.
```
s/toutes les données/toute ou partie des données/ ? Vu qu'on peut filtrer.

bboulesteix replied:
Tu as raison, je vais rajouter cette nuance dans le paragraphe. Merci bien. 👍

abulte commented on articles/_jeux-de-donnees/demander-a-datagouvfr-de-moisonner-votre-site.md:
```diff
@@ -0,0 +1,100 @@
+
+Choisissez ici le format des métadonnées associées aux jeux de données publiés sur votre plateforme. Ce format permet au moissonneur de savoir comment lire et interpréter vos métadonnées, pour bien les retranscrire sur data.gouv.fr.
+
+Pour les données de type ODS et CKAN, vous pouvez aussi ajouter des filtres, dans le but d’inclure ou d’exclure certains jeux de données du moissonnage automatique.
```
A développer ? Sur quoi on peut filtrer, comment...

abulte commented on articles/_jeux-de-donnees/demander-a-datagouvfr-de-moisonner-votre-site.md:
```diff
@@ -0,0 +1,100 @@
+
+Une fois votre moissonneur configuré, ce dernier passe en attente de validation. Il va être examiné par l’équipe en charge de data.gouv.fr, pour vérifier qu’il est bien réglé. Si c’est le cas, le moissonneur sera validé et vous recevrez une notification.
+
+Tant que votre moissonneur n’est pas validé, il ne référence aucun jeu de données sur data.gouv.fr.
```
A préciser je pense : on peut prévisualiser le résultat du moissonnage avant même que le moissonneur soit validé afin de vérifier que tout va bien.

bboulesteix replied:
Bonne idée, je vais ajouter un paragraphe sur la prévisualisation des résultats.

abulte commented on articles/_jeux-de-donnees/faire-une-recherche.md:
```diff
@@ -0,0 +1,119 @@
+
+* jeux de données ;
+* réutilisations ;
+* organisations.
```
* territoires

bboulesteix replied:
Tiens, je ne savais pas qu’il était possible de rechercher des départements ou des régions. Merci pour cette astuce @abulte. Ce serait vraiment bien de faire apparaître l’onglet `Territoires` tout le temps sur la page des résultats de recherche, même quand aucun territoire ne correspond au terme recherché — ça permettrait de savoir que cette catégorie existe. 🙂
<img width="461" alt="capture d ecran 2018-11-05 15 19 50" src="https://user-images.githubusercontent.com/3286488/48003402-a3e0a380-e10e-11e8-9b64-9a45203e77dc.png">

abulte replied:
En l'état je ne suis pas sûr qu'on ait envie de mettre la fonctionnalité Territoires en avant :trollface:

abulte commented on articles/_organisations/demander-a-rejoindre-une-organisation.md:
```diff
@@ -0,0 +1,29 @@
+---
+# Demander à rejoindre une organisation
+
+N’importe quel utilisateur peut demander à rejoindre n’importe quelle organisation. Si vous pensez que vous devriez figurer dans la liste des membres d’une organisation, faites une demande pour la rejoindre. Libre ensuite à l’administrateur de l’organisation en question d’accepter votre demande.
```
J'ai remarqué qu'il y pas mal de gens qui font des demandes "farfelues" pour rejoindre une organisation, par exemple un citoyen qui demande à rejoindre l'organisation de sa ville. Il faudrait peut-être préciser qu'appartenir à une organisation ne se justifie que si on veut publier/éditer/modérer des données en son nom.

abulte replied:
Je veux bien que tu traites ça ⬆️ si tu trouves le point pertinent et après LGTM.

abulte commented on articles/_organisations/demander-a-rejoindre-une-organisation.md:
```diff
@@ -0,0 +1,29 @@
+2. Rendez-vous sur la page de l’organisation que vous aimeriez rejoindre ;
+3. Cliquez sur le bouton **REJOINDRE** situé sous le logo de l’organisation ;
+4. Dans la fenêtre qui s’ouvre alors, expliquez ce qui motive votre demande puis cliquez sur **ENVOYER LA DEMANDE** ;
+5. Les administrateurs de l’organisation sont informés par e-mail que vous souhaitez intégrer leur équipe.
```
💡 

6. Si votre demande n'est pas acceptée au bout d'un certain temps, n'hésitez pas à contacter directement les administrateurs de votre organisation pour qu'ils acceptent votre demande.

Pas mal de gens restent bloqués à cette étape.

bboulesteix replied:
> N'hésitez pas à contacter directement les administrateurs de votre organisation

Je veux bien conseiller aux gens de faire ça, mais comment font-ils s’ils n’ont pas les coordonnées des administrateurs en question, @abulte ? 🤔 Cette solution ne marche pas dans le cas général, à mon avis — mais je rate peut-être quelque chose ici. Elle ne fonctionne que si la personne qui demande à rejoindre une organisation connaît _personnellement_ son administrateur — ce qui doit souvent être le cas, je te l’accorde, mais pas toujours le cas.

Accessoirement, la solution que tu proposes en guise de sixième point revient aussi à demander aux utilisateurs de ne pas se fier aux notifications envoyées par data.gouv.fr et de contourner le produit, ce qui me gène un peu, même s’il faut être pragmatique sur ce point et accepter que certaines notifications passent à l’as.

abulte replied:
Effectivement. C’est un pis-aller en attendant qu’on puisse relancer depuis le site par exemple. Mais on peut s’en passer.

bboulesteix replied:
Merci pour ta réponse. 👍 Je ferme la discussion pour le moment. Nous serons peut-être amenés à reparler de ce point si le sujet refait surface dans les remarques des utilisateurs. 🙂

abulte commented on articles/_organisations/differences-administrateur-editeur-organisation.md:
```diff
@@ -0,0 +1,25 @@
+Au sein d’une organisation, un éditeur peut :
+
+* publier des jeux de données au nom de l’organisation ;
+* référence des réutilisations au nom de l’organisation ;
```
s/référence/référencer/

bboulesteix replied:
Merci. Corrigé. ✅

abulte commented on articles/_organisations/differences-administrateur-editeur-organisation.md:
```diff
@@ -0,0 +1,25 @@
+Un administrateur possède les mêmes droits qu’un éditeur, mais il peut, en plus :
+
+* ajouter des utilisateurs à l’organisation ;
+* accepter ou valider les demandes d’adhésion à l’organisation.
```
* supprimer l'organisation ?

bboulesteix replied:
Bien vu, bon point. Je viens d’ajouter cette permission dans l’article.

abulte commented on articles/_organisations/retirer-un-utilisateur-d-une-organisation.md:
```diff
@@ -0,0 +1,20 @@
+3. Allez sur la page de suivi de l’organisation à laquelle vous souhaitez ajouter un membre, en cliquant sur le nom de votre organisation, dans la colonne de gauche ;
+4. Dans le bloc **Membres**, sur la droite de votre écran, cliquez sur le nom du membre que vous souhaitez retirer ;
+5. Dans la fenêtre qui s’ouvre alors, cliquez sur **Supprimer** ;
+6. Rafraichissez votre page pour constater le retrait du membre.
```
👌

bboulesteix replied:
Ahah, je n’ai pas trouvé mieux que la méthode évoquée dans le sixième point pour constater le changement (retrait d’un utilisateur). S’il est possible de constater le retrait d’un utilisateur sans rafraichir la page, ça m’intéresse. 👀

abulte replied:
Non pas mieux ;-) Sad but true.

abulte commented on articles/_reutilisations-et-discussions/moderer-une-discussion.md:
```diff
@@ -0,0 +1,72 @@
+7. Repérez la discussion à clore et cliquez sur son titre ;
+8. Dans la fenêtre qui s’ouvre alors, saisissez un commentaire (par exemple : _Je ferme la discussion car elle me semble terminée._ ), puis cliquez sur **Commenter et clore la discussion**.
+
+## Supprimer tout un fil de discussion
```
Il me semblait que seul les super-admins pouvaient faire ça 🤔 Mais j'imagine que tu as dû tester 👌.

bboulesteix replied:
> Il me semblait que seul les super-admins pouvaient faire ça.

Tu as raison. Seuls les « super admins » peuvent faire ça. 🙂 Je crois que j’ai encore un peu de mal de mon côté pour faire la distinction entre administrateur ordinaire et _super admin_. Je supprime le paragraphe en question de l’article, donc.

Commit:
Modifie demander-a-datagouvfr-de-moisonner-votre-site.md

Pour préciser que le format MAAF est spécifique au ministère de l’agriculture.

Commit:
Modifie demander-a-datagouvfr-de-moisonner-votre-site.md

Ajoute une phrase sur la possibilité de n’importer que certains jeux de données.

Commit:
Modifie faire-une-recherche.md

Ajoute la notion de territoire dans les objets qui peuvent être recherchés sur data.gouv.fr.

Commit:
Modifie differences-administrateur-editeur-organisation.md

Corrige une coquille.

Commit:
Modifie moderer-une-discussion.md

Biffe la section de l’article consacrée à la suppression d’une discussion, car cette fonctionnalité n’est proposée qu’aux « super » administrateurs, c’est-à-dire aux membres d’Etalab qui travaillent sur data.gouv.fr.

Commit:
Modifie differences-administrateur-editeur-organisation.md

Ajoute une permission dans la section consacrée aux administrateurs.

Commit:
Modifie demander-a-datagouvfr-de-moisonner-votre-site.md

Ajoute un paragraphe sur le test des moissonneurs fraichement créés.

Commit:
Modifie demander-a-datagouvfr-de-moisonner-votre-site.md

Ajoute des précisions sur les filtres des moissonneurs.
